com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -1348,10 +1348,10 @@ private void HandleRawTransportPoll(NetworkEvent networkEvent, ulong clientId, A
 #if DEVELOPMENT_BUILD || UNITY_EDITOR
                     s_TransportDisconnect.Begin();
 #endif
-                    OnClientDisconnectCallback?.Invoke(clientId);
-
                     clientId = TransportIdToClientId(clientId);
```

Comment:
I really dislike `clientId = TransportIdToClientId(clientId);`. Yeah, I realize it's not your change. But if we're playing in this, can we do:
`var clientId = TransportIdToClientId(transportId);` and rename `clientId` to `transportId` above (or just rename the input param, we already have transportId) ? The variable changing semantics mid-way is a recipe for future bugs.

Then again, it's a back-port, so maybe it's not worth it, or not the right place and this would go in `develop`.